Document important behavior of link="preload" (#19598)
mdn/content: files/en-us/web/html/link_types/preload/index.md

The `preload` value of the {{htmlelement("link")}} element's {{htmlattrxref("rel", "link")}} attribute lets you declare fetch requests in the
HTML's {{htmlelement("head")}}, specifying resources that your page will need very soon, which you want to start loading early in the page lifecycle,

> before browsers' main rendering machinery kicks in. This ensures they are available earlier and are less likely to block the page's render, improving performance. Even though the name contains "load", it does not actually load(or execute) the script, but only fetches and caches it with high priority.

Reviewer: What about:

```suggestion
before browsers' main rendering machinery kicks in. This ensures they are available earlier and are less likely to block the page's render, improving performance. Even though the name contains the term _load_, it doesn't load and execute the script but only schedules it to be downloaded and cached with a higher priority.
```

Author: Yes, looks good to me.
